src/components/partnership/dto/create-partnership.dto.ts

```diff
@@ -0,0 +1,43 @@
+import { Type } from 'class-transformer';
+import { ValidateNested } from 'class-validator';
+import { DateTime } from 'luxon';
+import { Field, ID, InputType, ObjectType } from 'type-graphql';
+import { DateField } from '../../../common';
+import { PartnershipAgreementStatus } from './partnership-agreement-status.enum';
+import { PartnershipType } from './partnership-type.enum';
+import { Partnership } from './partnership.dto';
+
+@InputType()
+export class CreatePartnership {
+  @Field(() => PartnershipAgreementStatus)
+  readonly agreementStatus: PartnershipAgreementStatus;
+
+  @Field(() => PartnershipAgreementStatus)
+  readonly mouStatus: PartnershipAgreementStatus;
+
+  @DateField({ nullable: true })
+  readonly mouStart: DateTime | null;
+
+  @DateField({ nullable: true })
+  readonly mouEnd: DateTime | null;
+
+  @Field(() => ID)
+  readonly organizationId: string;
+
+  @Field(() => [PartnershipType])
+  readonly types: PartnershipType[];
```

Comment:
I'm not sure if I missed this before, all of these should be optional (with `?:`) except `organizationId`.

Reply:
Done!

Reply:
Decorators need `{ nullable: true }` too 😬

Reply:
Whoops! 😆 Okay should be good now. Just fixed the type error that was failing in CI as well